### ISSUE TITLE
Contacts not allowed above 6 connection even in career mode

### ISSUE THREAD

**Issue by chummer5a** [chummer5a/chummer5a#90]
<a href="https://github.com/GoogleCodeExporter"><img src="https://avatars.githubusercontent.com/u/9614759?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [GoogleCodeExporter](https://github.com/GoogleCodeExporter)**
_Tuesday Mar 17, 2015 at 08:56 GMT_
_Originally opened as https://github.com/chummer5a/chummer5/issues/16_

----

```
reference SR5 386


```

Original issue reported on code.google.com by `fallen.t...@gmail.com` on 30 Sep 2014 at 1:43

**Comment by chummer5a** [comment]
<a href="https://github.com/GoogleCodeExporter"><img src="https://avatars.githubusercontent.com/u/9614759?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [GoogleCodeExporter](https://github.com/GoogleCodeExporter)**
_Tuesday Mar 17, 2015 at 08:56 GMT_

----

```
Fix would be in ContactControl.Designer.cs

Change line 55 from 6 to 12.
```

Original comment by `fallen.t...@gmail.com` on 20 Oct 2014 at 2:38

**Comment by chummer5a** [comment]
<a href="https://github.com/GoogleCodeExporter"><img src="https://avatars.githubusercontent.com/u/9614759?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [GoogleCodeExporter](https://github.com/GoogleCodeExporter)**
_Tuesday Mar 17, 2015 at 08:56 GMT_

----

```
hey Fallen, where do I find that file? I'm new to coding and I can't seem to 
figure out where it's located...

Thanks
```

Original comment by `DonovanA...@gmail.com` on 21 Nov 2014 at 5:09

**Comment by chummer5a** [comment]
<a href="https://github.com/GoogleCodeExporter"><img src="https://avatars.githubusercontent.com/u/9614759?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [GoogleCodeExporter](https://github.com/GoogleCodeExporter)**
_Tuesday Mar 17, 2015 at 08:56 GMT_

----

```
it's located in trunk->Chummer

http://i.imgur.com/dwlB4N0.png
```

Original comment by `fallen.t...@gmail.com` on 21 Nov 2014 at 6:59

**Comment by chummer5a** [comment]
Resolved in 5.159.
